modules/vaos/app/services/vaos/v1/fhir_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../base_service'
+require 'common/exceptions/external/backend_service_exception'
+
+module VAOS::V1
+  class FHIRService < VAOS::BaseService
+    STATSD_KEY_PREFIX = 'api.vaos.fhir'
+
+    def perform(http_method, path, params = nil)
```

Comment:
I originally had this method named `make_request` as `perform` is private in the parent class. Don't know if that's more or less confusing than having it named the same thing. This way kind of breaks encapsulation but in this instance it's meant to be called with dynamic http methods and paths.

Reply:
I'd also like to protect it with a hash of resources (paths), maybe we can load and cache the conformance doc and confirm that resource is available?

Reply:
Actually since we're not doing dynamic routes having a hash of resources probably isn't necessary.

Reply:
might prove useful in route globbing ruleset to have, but that should only be needed temporarily.

Reply:
you could do 

private_method :perform if you don't want to  break encapsulation after defining the method.

Reply:
but what's wrong with the existing perform method? just need distinct headers added in right?